src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(
+    item => item.title.toLowerCase().includes(query.toLowerCase())
+    || item.description.toLowerCase().includes(query.toLowerCase()),
```

Comment:
Transform query to lower case in one place instead of doing it here twice. DRY - don't repeat yourself.